DU0038/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using DU0038.Controller;
+
+namespace DU0038
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var programController = new ProgramController();
```

Comment:
Itt a konstruktorod két dolgot csinál: Létrehozza a kontroller objektumot az adattagjaival **ÉS** futtatja az alkalmazás logikát. Az SRP metódusokra/konstruktrorokra is alkalmazható ám. Pl az alkalmazás futtatást ki lehetett volna szervezni egy `Run()` metódusba.